pywinauto/win32structures.py

```diff
@@ -51,41 +51,41 @@ def __str__(self):
 
         fields in exceptList will not be printed"""
         lines = []
-        for f in self._fields_:
+        for f in getattr(self, "_fields_", list()):

```

Comment:
This is minor, but I'd suggest to use `[]` instead of `list()` as it aligns with the rest of library code.